build/vsts-ci.yml

```diff
@@ -155,6 +159,10 @@ phases:
       artifactName: PackageAssets
       artifactType: container
 
+  # Terminate all dotnet build processes.
+  - script: $(Build.SourcesDirectory)/Tools/dotnetcli/dotnet.exe build-server shutdown
+    displayName: Dotnet Server Shutdown
+
 ################################################################################
 - phase: Package
```

Comment:
You may also need/want to call `dotnet.exe build-server shutdown` in the `Package` step.